test/lib/localStorage.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * A simple localStorage implementation using in-memory storage.
+ */
+export default class LocalStorage {
```

Comment:
why `export default` ? instead of `export class` ?

Reply:
I agree, `export class` makes it easier to add other imports in the future and makes the importing-side clearer too